crypto3/libs/hash/include/nil/crypto3/hash/detail/merkle_damgard_construction.hpp

```diff
@@ -151,12 +155,15 @@ namespace nil {
                     using namespace nil::crypto3::detail;
 
                     std::array<length_type, 1> length_array = {{length}};
-                    std::array<word_type, length_words> length_words_array;
-                    pack<endian_type, endian_type, length_bits, word_bits>(length_array.begin(), length_array.end(),
-                                                                           length_words_array.begin());
-                    // Append length
-                    for (std::size_t i = length_words; i; --i)
-                        block[block_words - i] = length_words_array[length_words - i];
+                    std::array<word_type, length_type_bits / word_bits> length_words_array;
```

Comment:
```suggestion
                    std::array<word_type, length_words> length_words_array;
```

Reply:
No, length_words is different, can't use it here.